### PR TITLE
Add documentation for cloning

### PR DESCRIPTION
**Pull request by nolancon**

### CODE REVIEW

File: docs/concepts/cloning.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+The Ondat Cloning feature adds support for specifying existing PVCs in the dataSource field to indicate a user would like to clone a Volume.
+
+### What is a Clone?
+
+A **clone** is defined as a duplicate of an existing Kubernetes Volume that can be consumed as any standard Volume would be.
+
+### Why use Cloning:
+
+TODO.
```
Comment (nolancon) [inline docs/concepts/cloning.md:19]: Someone from CS may have some concrete use cases for here?

**Comment by Mojachieee** [comment]
@nolancon can you target this at the branch `2.10` (you may need to create this).

File: docs/concepts/cloning.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+title: "Ondat Cloning"
+linkTitle: "Ondat Cloning"
+weight: 1
+---
+
+## Overview
+
+> 💡 This feature is available in release `v2.10.0` or greater.
+
+The Ondat Cloning feature adds support for specifying existing PVCs in the dataSource field to indicate a user would like to clone a volume.
```
Comment (Ricardo-Osorio) [inline docs/concepts/cloning.md:11]: This sentence hurts my brain.

Hoping someone else pitches in but I think this section should only introduce the feature, so something like
"Cloning creates a new volume by coping the data from an existing one"

Then you talk about the "dataSource" on the section of how to use it

File: docs/operations/how-to-clone-a-volume.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
+  dataSource:
+    kind: PersistentVolumeClaim
+    name: pvc-1
+```
+
+The result is a new Volume with the name clone-of-pvc-1 that has the exact same content as the specified source Volume `pvc-1`.
```
Comment (Ricardo-Osorio) [inline docs/operations/how-to-clone-a-volume.md:28]: ~The result is a new Volume with the name clone-of-pvc-1 that has the exact same content as the specified source Volume `pvc-1`.~
This results in a new volume named "clone-of-pvc-1" containing a copy of the data of source volume "pvc-1".

File: docs/operations/how-to-clone-a-volume.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+title: "How To Clone a Volume"
+linkTitle: How To Clone a Volume
+---
+
+## Create a Clone of an Existing Volume
+
+The Kubernetes [Volume Cloning](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/) feature provides users with a mechanism to create a clone Volume by adding a dataSource that references an existing Volume.
```
Comment (Ricardo-Osorio) [inline docs/operations/how-to-clone-a-volume.md:8]: No need to explain it again here, it's already done on the /docs/concepts/cloning.md
We can add a link to that page

~The Kubernetes [Volume Cloning](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/) feature provides users with a mechanism to create a clone Volume by adding a dataSource that references an existing Volume.~

[Cloning](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/) a volume is done by referring to an existing one in the "dataSource" field of the manifest.

File: docs/concepts/cloning.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+### Why use Cloning
+
+TODO.
+
+### How Does It Work?
+
+The Kubernetes [Volume Cloning](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/) feature provides users with a mechanism to create a clone volume from an existing source volume.
+
+For more information, see [How to Clone a Volume](/docs/operations/how-to-clone-a-volume).
```
Comment (Ricardo-Osorio) [inline docs/concepts/cloning.md:25]: ~For more information~

For an example

File: docs/concepts/cloning.md (hunk 1 above)
Comment (chris-milsted) [reply docs/concepts/cloning.md:19]: Cloning is very useful when you need to copy the data from one persistent volume to another. For example there could be an application error where an issue is caused by a data quality issue. Being able to copy the contents of a volume and debug this without affecting the running application can be extremely useful when attempting to fix such issues.

**Pull request merged**